contributed_definitions/nyaml/NXmpes.yaml

```diff
@@ -221,15 +491,18 @@ NXmpes(NXobject):
           other metadata file. In the case these are not available, free-text description.
       temperature(NX_FLOAT):
         unit: NX_TEMPERATURE
+        exists: recommended
         doc: |
           In the case of a fixed temperature measurement this is the scalar temperature of
           the sample. In the case of an experiment in which the temperature is changed and
           recoded, this is an array of length m of temperatures. This should be a link to
           /entry/instrument/manipulator/sample_temperature.
       situation:
+        exists: optional
         enumeration: [vacuum, inert atmosphere, oxidising atmosphere, reducing atmosphere]
       gas_pressure(NX_FLOAT):
         unit: NX_PRESSURE
+        exists: optional
```

Comment:
I would suggest recommended for these two

Reply:
We anyway planned to describe these fields by NXenvironment (see https://github.com/FAIRmat-NFDI/nexus_definitions/pull/68) and there, they would be recommended. We can set them here to recommended as well until the other PR is merged into this one.

Reply:
Please do so and resolve

Reply:
Done